source/locale/pt_BR/LC_MESSAGES/flow.po

```diff
@@ -865,125 +864,126 @@ msgstr ""
-"qual tenant o fluxo pertence) onde as mensagens são produzidas"
-"com informações sobre o nome do arquivo, formato de codificação e conteúdo do arquivo."
+"Publica no tópico ``tenant.dojot.ftp`` (*tenant* é definido por qual "
+"tenant o fluxo pertence) onde as mensagens são produzidas com informações"
+" sobre o nome do arquivo, formato de codificação e conteúdo do arquivo."
 
-#: ../../source/flow.rst:623
-msgid ": Publish in FTP topic configuration"
-msgstr ": Configurações do Publicar no tópico FTP "
+#: ../../source/flow.rst:622
+msgid ": *Publish in FTP topic* configuration"
+msgstr ": Configurações do *Publicar no tópico FTP* "
 
-#: ../../source/flow.rst:627
+#: ../../source/flow.rst:626
 msgid ""
 "**Encoding** *(required)*: Encoding the contents of the file to be sent. "
 "Valid values are: ascii, base64, hex, utf16le, utf8 and binary."
 msgstr ""
-"**Codificação** *(obrigatório)*: Codificação do conteúdo do arquivo a ser enviado. "
-"Os valores válidos são: ascii, base64, hex, utf16le, utf8 e binary."
+"**Codificação** *(obrigatório)*: Codificação do conteúdo do arquivo a ser"
+" enviado. Os valores válidos são: ascii, base64, hex, utf16le, utf8 e "
+"binary."
 
-#: ../../source/flow.rst:628
+#: ../../source/flow.rst:627
 msgid "**Filename** *(required)*: Variable with the name of the file to be sent."
 msgstr ""
 "**Nome do arquivo (Filename)** *(obrigatório)*: variável que contém o "
 "nome do arquivo a ser enviado."
 
-#: ../../source/flow.rst:629
+#: ../../source/flow.rst:628
 msgid "**Content** *(required)*: Variable with the file contents to be sent."
 msgstr ""
-"**Conteúdo do arquivo (Content)** *(obrigatório)*: Essa variável "
-"deve conter o conteúdo do arquivo a ser enviado."
+"**Conteúdo do arquivo (Content)** *(obrigatório)*: Essa variável deve "
+"conter o conteúdo do arquivo a ser enviado."
 
-#: ../../source/flow.rst:633
+#: ../../source/flow.rst:632
 msgid "Example of message sent by this node below:"
 msgstr "Exemplo de mensagem enviada por este nó abaixo:"
 
-#: ../../source/flow.rst:651
+#: ../../source/flow.rst:650
 msgid "Where the keys above are:"
 msgstr "Onde as chaves acima são:"
 
-#: ../../source/flow.rst:653
+#: ../../source/flow.rst:652
 msgid ""
 "msgId:  Value of type uuidv4 used to uniquely identify the message in the"
 " context of dojot."
 msgstr ""
-"msgId: Valor do tipo uuidv4 usado para identificar unicamente a mensagem no "
-"contexto da dojot."
+"msgId: Valor do tipo uuidv4 usado para identificar unicamente a mensagem "
+"no contexto da dojot."
 
-#: ../../source/flow.rst:654
+#: ../../source/flow.rst:653
 msgid ""
-"ts: Timestamp in Unix Timestamp (ms) format from the moment the message was "
-"produced."
+"ts: Timestamp in Unix Timestamp (ms) format from the moment the message "
+"was produced."
 msgstr ""
-"ts: Timestamp no formato Unix Timestamp (ms) do momento em que a mensagem foi"
-"produzido."
+"ts: Timestamp no formato Unix Timestamp (ms) do momento em que a mensagem"
+" foiproduzido."
```

Comment:
The translation tool automatically changes the number of words per line and sometimes joins the words together. :laughing:

I think this happened because in the diff with the old version (:point_left:) there was no space at the end of the line (before closing the quotes).